docs/components/buttons/elements.md

```diff
@@ -1,89 +1,89 @@
 ```
 
-### Secondary
+#### Secondary
 The secondary button is for secondary or tertiary actions. Can be combined with "small" and "quiet" attributes. "secondary" is the default value of the "variant" attribute.
 
-```js
+```html
 <w-button variant="secondary">Secondary button</w-button>
 <w-button variant="secondary" small>Secondary button small</w-button>
 ```
 
-### Negative
+#### Negative
 The negative button is for emphasizing actions that can be destructive or have negative consequences if taken. Can be combined with "small" and "quiet" attributes.
 
-```js
+```html
 <w-button variant="negative">Negative button</w-button>
 <w-button variant="negative" small>Negative button small</w-button>
 ```
 
-### Utility
+#### Utility
 
-```js
+```html
 <w-button variant="utility">Utility button</w-button>
 <w-button variant="utility" small>Utility button small</w-button>
 ```
 
-### Link
+#### Link
 
-```js
+```html
 <w-button variant="link">Link button</w-button>
 <w-button variant="link" small>Link button small</w-button>
 ```
 
-### Pill
+#### Pill
 
-```js
+```html
 <w-button variant="pill">💙</w-button>
 <w-button variant="pill" small>💙</w-button>
 ```
 
-## Quiet
+#### Quiet
 By default, buttons have a visible stroke and fill. Quiet buttons do not have this visible stroke or fill and should only be used for secondary actions within a button group. Quiet buttons have the same padding rules as other action buttons and reveal a button-like background color when hovered.
-```js
+```html
 <w-button quiet="">Quiet secondary button</w-button>
 <w-button quiet="" variant="negative">Quiet negative button</w-button>
 <w-button quiet="" variant="utility">Quiet utility button</w-button>
 ```
 
-## Link
+#### Link
 Buttons will be rendered as an anchor (a tag) if they use an href attribute.
-```js
+```html
 <w-button href="https://google.no">Button as anchor</w-button>
 ```
 
-## Loading state
+#### Loading state
 To show the user that the action they triggered has begun, buttons have an in progress or loading state.
-```js
+```html
 <w-button variant="primary" loading="">Primary button loading</w-button>
 ```
-## Full width
+#### Full width
 
 The fullWidth prop sets the button's width to its parent's width. 
-```js
- <div class="w-full">
-    <w-button full-width variant="primary">Primary button full width</w-button>
+```html
+<div class="w-full">
+  <w-button full-width variant="primary">Primary button full width</w-button>
 </div>
 ```
 
-## Disabled state
+#### Disabled state
 
-Disabled is an anti-pattern and is not supported. There will ALWAYS be users who don't understand why an element is disabled, or users who can't even see that it is disabled because of poor lighting conditions or other reasons.
+Disabled is an anti-pattern and is not supported.
+There will ALWAYS be users who don't understand why an element is disabled, or users who can't even see that it is disabled because of poor lighting conditions or other reasons.
```

Comment:
I don't think it's necessary to emphasise the ALWAYS if we aim at the documentation language to be neutral 😉

Reply:
Agree. Even though it wasn't something added here. I'll fix. 👍

Reply:
Ah, sorry, I didn't notice it in the diff 😅